.circleci/config.yml

```diff
@@ -62,8 +62,13 @@ jobs:
           environment:
             JAVA_HOME: /usr/lib/jvm/default-java
           command: |
-            make rsthtml
+            make rsthtml SPHINXOPTS="-b html -W --keep-going -n -T"
```

Comment:
Can we include these options in `Makefile`?

https://github.com/mlflow/mlflow/blob/d6d4d63fbaf1a98b70c450c49658a56fcb3f55b8/docs/Makefile#L5

Reply:
definitely. I completely forgot about that config setup.